ToDoList/src/main/java/ua/goit/timonov/enterprise/module_8/UpdateServlet.java

```diff
@@ -0,0 +1,49 @@
+package ua.goit.timonov.enterprise.module_8;
+
+import org.apache.commons.lang3.StringUtils;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+import java.util.Enumeration;
+
+/**
+ * Servlet for update ToDoList form
+ */
+public class UpdateServlet extends HttpServlet {
+
+    public static final String ENABLED_TASK = "Enabled Task=";
+    public static final String DELETED_TASK = "Deleted Task=";
+
+    // adds to session new attribute from request, removes from session attributes marked in request to delete
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        HttpSession session = request.getSession();
+        String parameter = request.getParameter("taskName");
+        if (isValid(parameter)) {
+            String attributeName = ENABLED_TASK + parameter;
+            String attributeObject = request.getParameter("categoryName");
+            session.setAttribute(attributeName, attributeObject);
+        }
+
+        Enumeration<String> parameterNames = request.getParameterNames();
```

Comment:
But method getParameterNames() from javax.servlet.ServletRequest returns Enumeration<String>.
Fixed iteration as it suggested in Javadoc for java.util.Enumeration<E>:
`for (Enumeration<String> parameterNames = request.getParameterNames(); parameterNames.hasMoreElements(); ) {`